lua/neogit/lib/git/stash.lua

```diff
@@ -107,6 +109,13 @@ function M.list()
   return cli.stash.args("list").call({ hidden = true }).stdout
 end
 
+function M.rename(stash)
+  local message = input.get_user_input("New name")
+  local oid = rev_parse.abbreviate_commit(stash)
+  cli.stash.drop.args(stash).call()
+  cli.stash.store.message(message).args(oid).call()
```

Comment:
Oh, wild, so to rename is a drop + store with message? Cool

Reply:
Didn't know that myself, taken from [here](https://stackoverflow.com/questions/25931026/how-can-i-rename-a-git-stash)